library/core/src/num/int_macros.rs

```diff
@@ -668,6 +668,42 @@ macro_rules! int_impl {
             }
         }
 
+        /// Checked integer division. Computes `self / rhs`, returning `None` if one `rhs == 0`,
+        /// the division results in overflow, or remainder is not zero.
```

Comment:
>Division can't overflow, right?

For signed integers, unfortunately, it can. When you divide `MIN` with `-1`, the result will be bigger than `MAX`.

Reply:
Gotcha! If we describe division overflow anywhere else in the standard lib docs, we should perhaps make "overflow" a link to that place.

Reply:
Here I followed docs of `checked_div`, `checked_div_euclid`, `checked_rem`, and `checked_rem_euclid`. It's probably better to add such link (if it exists) for all these methods in a separate PR.